docs/content/development/extensions-contrib/tdigestsketch-quantiles.md

```diff
@@ -29,16 +29,13 @@ T-Digest (https://github.com/tdunning/t-digest) is a popular datastructure for a
 rank-based statistics such as quantiles and trimmed means.
 The datastructure is also designed for parallel programming use cases like distributed aggregations or map reduce jobs by making combining two intermediate t-digests easy and efficient.
 
-There are three flavors of T-Digest sketch aggregator available in Apache Druid (incubating):
+There are two flavors of T-Digest sketch aggregator available in Apache Druid (incubating):
 
-1. buildTDigestSketch - used for building T-Digest sketches from raw numeric values. It generally makes sense to
-use this aggregator when ingesting raw data into Druid. One can also use this aggregator during query time too to
-generate sketches, just that one would be building these sketches on every query execution instead of building them
-once during ingestion.
-2. mergeTDigestSketch - used for merging pre-built T-Digest sketches. This aggregator is generally used during
-query time to combine sketches generated by buildTDigestSketch aggregator.
-3. quantilesFromTDigestSketch - used for generating quantiles from T-Digest sketches. This aggregator is generally used
-during query time to generate quantiles from sketches built using the above two sketch generating aggregators.
+1. buildTDigestSketch - It generally makes sense to use this aggregator when ingesting raw data into Druid to
+generate pre-agrgegated sketches. One can also use this aggregator during query time too to combine ingested T-Digest
+sketches.
+2. quantilesFromTDigestSketch - used for generating quantiles from T-Digest sketches. This aggregator is generally used
```

Comment:
Suggest rewording this section, there's only one aggregator since the PostAggregators don't really aggregate (they only operate on values within a single row), they're more like "post aggregation transforms".